tests/arch/arm/arm_irq_advanced_features/src/arm_dynamic_direct_interrupts.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ztest.h>
+#include <arch/cpu.h>
+#include <arch/arm/cortex_m/cmsis.h>
+
+
+void arm_direct_isr_handler_0(void *args)
+{
+  ARG_UNUSED(args);
+
+	printk("test\n");
```

Comment:
Why not set a variable and then check that it is set in the test main function?

Reply:
Work in progress - stand by!